src/abstractions/Changelog-old.md

```diff
@@ -0,0 +1,449 @@
+# Changelog (old)
+
+All notable changes to this project will be documented in this file.
```

Comment:
To confirm, what you mean here is 
- remove the old changelog of the abstractions (we have the releases page present)
- keep the old changelog for the other packages (their history is in other repos)

> what do you think about deleting this file since we already have the content in the releases?

I can't say for sure what the right thing to here with regards to user preference of going over a file vs navigating the releases page on GitHub. I think keeping them both doesn't hurt and is an easily reversible decision (we can simply delete the file as we don't expect changes in it going forward). 
I'm okay with changing this though if you have strong preference here..

Reply:
I don't have a strong preference either we can keep it. Resolving.